packages/vectra_rux/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.3.2"
+  changes:
+    - description: Fix CEL code to properly handle scientific notation in `from` query parameter.
```

```suggestion
    - description: Fix CEL code to properly handle `from` query parameter.
```
The notion of scientific notation here is not informative to users and the issue here is the result of an implementation detail, rather than an intrinsic aspect of the system.